ci/jjb/jobs/projects.yaml

```diff
@@ -21,11 +21,11 @@
     os:
         - 'f24'
         - 'f25'
+        - 'f26'
         - 'rhel7'
     pulp_version:
-        - '2.15':
+        - 'master':
```

Comment:
This change will break the coverage report functionality in `pulp-dev.yaml`. Search that file for `PULP_VERSION` and you'll see what I mean.

I don't think that the coverage report works that well right now, and I'm not aware of anyone that uses it. Still, it'd be good to either fix or drop that bit of code.

Reply:
Since this is for pulp-packaging, it should be ok (the pulp-dev job definition still has this setting set appropriately)

Reply:
Sorry. I referenced the file. I think this change will break the coverage report defined in `pulp-packaging-build.yaml`.